### DISCUSSION TITLE
how to run the specific test scenario in a specification file repeatedly

### DISCUSSION THREAD

**Discussion by skulkarni97** [getgauge/gauge#2495] [General]
how to run the specific test scenario in a specification file repeatedly?

example : i have a test scenario which has to be repeated 20 times to create test data then how do i run it using gauge ...
any tags? can someone help me here.

**Comment by Elwis28** [comment]
I think my best bet is to use the table drive scenario: https://docs.gauge.org/writing-specifications?os=windows&language=javascript&ide=vscode#table-driven-scenario. 
If it not suits your needs, then i think you should create a new step where you put your test data generation code behid the step and then execute it.
